scripts/publish.sh

```diff
@@ -30,5 +45,12 @@ rm public/documentation/index.xml
 rm public/community/index.xml
 rm public/development/index.xml
 
+DESTINATION_STABLE_VERSION="public/documentation/stable"
+echo "Copying the stable documentation from ${STABLE_VERSION_FOLDER} to ${DESTINATION_STABLE_VERSION}"
+cp -r ${STABLE_VERSION_FOLDER} ${DESTINATION_STABLE_VERSION}
+
+echo "Add noindex meta tag to all HTML files in public/documentation/[0-9]*"
+find public/documentation/ -type f -regex "public/documentation/[0-9].*" -name "*.html" | xargs  sed -i '/<\/head>/i\<meta name="robots" content="noindex" \/>'
```

Comment:
The version in the current commit works for me in the git-bash (mingw64, which is coming with the latest git client binaries).

One thing alternative would also be to use `--in-place` instead of `-i`, as the optional suffix would have to be provided as `--in-place=backup` - that shouldn't confuse the different shell interpreters as much as the `-i` variant.